unitary/examples/quantum_rpg/npcs_test.py

```diff
@@ -86,3 +86,23 @@ def test_purple_foam():
     assert msg == "PurpleFoam bubbles measures person_1 as HURT."
     msg = qar.act_on_enemy_qubit(c.get_hp("person_1"), 0.3, slime=0.25)
     assert msg == "PurpleFoam bubbles covers person_1 with foam!"
+
+
+def test_schrodinger_cat():
```

Comment:
Would be nice to see a test of act_on_enemy_qubit().

Reply:
Done.  Also added a quantopedia entry now that the quantopedia is committed.